.travis.yml

```diff
@@ -20,6 +20,8 @@ matrix:
     env: JAVA_VERSION=11 DEPLOY=true
   - jdk: openjdk16
     env: JAVA_VERSION=16
+  - jdk: openjdk17
+    env: JAVA_VERSION=17
```

Comment:
Unfortunately this does not work because Travis CI didn't run since June 6, 2021, see https://travis-ci.org/github/vavr-io/vavr Perhaps it's time to move to GitHub actions 😢 Here is what I did few days ago for vavr-jackson https://github.com/vavr-io/vavr-jackson/pull/195

Reply:
I will try to move to GitHub actions :)

Reply:
Good point! Yep, GH Actions is the way to go!